Actions/.Modules/ReadSettings.psm1

```diff
@@ -283,6 +283,8 @@ function GetDefaultSettings
         The value of the current GitHub environment settings variable, based on workflow context. Default is $ENV:ALGoEnvSettings.
     .PARAMETER environmentName
         The value of the environment name, based on the workflow context. Default is $ENV:ALGoEnvName.
+    .PARAMETER customSettings
+        Custom settings JSON string that will be applied last to override any other settings. These settings have the highest precedence.
```

Comment:
Minor thing;
```suggestion
        JSON formatted string that will be applied last to override any other settings. These settings have the highest precedence.
```

Could you update the description of this function too? It says something like "The settings are read from the following files:" and then it lists the places where settings are applied from.